cli-commands/compiler/etherlime-compile/profiler.js

```diff
@@ -30,8 +31,15 @@ let prepareFiles = async (options) => {
   let sourceFilesArtifacts = {};
   try {
     let files = await getFiles(options);
-    files.forEach(function (sourceFile) {
-      sourceFilesArtifacts[sourceFile] = [];
+
+    if(files.vyperFiles && files.vyperFiles.length > 0) {
+      await vyperCompiler(files.vyperFiles, options.contracts_build_directory)
+    }
+
+    let solFiles = files.solFiles || files
```

Comment:
What is the case where there are no solFiles defined? Shouldn't getFiles always returns solFiles array (regardless that it sometimes can be empty)